AUTHORS

```diff
@@ -26,6 +27,7 @@ Kevin Chirls <kchirls@users.noreply.github.com>
 Liu DongMiao <liudongmiao@gmail.com>
 Luan Nico <luannico27@gmail.com>
 Maarten Mulders <mthmulders@users.noreply.github.com>
+Manu Sridharan <manu@cs.ucr.edu>
```

Comment:
@msridhar Let me know if you'd like another email listed.

Reply:
Could we possibly use `msridhar@gmail.com`?  Thanks a lot!